src/controllers/funds.js

```diff
@@ -0,0 +1,28 @@
+const Fund = require("../models/fund")
+
+const createFund = async (req, res) => {
+    try {
+        const fund = await Fund.create({
+            publisherId: req.user.id,
```

Comment:
Use distructuring instead of using `req.user.id` or `req.body.title` directly. Ex: 
```
const { title, content } = req.body
```